frontend/ui/widget/menu.lua

```diff
@@ -167,18 +167,17 @@ function MenuItem:init()
     end
 
     -- State button and indentation for tree expand/collapse (for TOC)
-    local state_button_width = self.state_size.w or 0
-    local state_button = self.state or HorizontalSpan:new{
-        width = state_button_width,
+    local state_button = self.state or HorizontalSpan:new{}
+    local state_indent = self.table.indent or ""
```

Comment:
in `self.table.indent ` , self.table feels like a typo, there is no other self.table.

Reply:
we pass `table = self.item_table[i],` to MenuItem. so i did not feel a need to create a new indent parameter.
https://github.com/koreader/koreader/blob/7783d9c8e37099fa2e138d660dc76ed4ec5257ca/frontend/ui/widget/menu.lua#L1085

Reply:
OK, I see.
That was previous code, so not yours, but it's stupid to name that thing "table", and not having it documented in `local MenuItem = InputContainer:new{` made it harder to discover :/